### PR TITLE
make number validation screen

### PR DESCRIPTION
**Pull request by Harrisali93**
## Jira Story Link:
[KAS-7](https://leezeit.atlassian.net/browse/KAS-7)

**Description**:
Verification screen with one logo, two buttons, three labels and four textfields.
Buttons are clickable but do not perform any action.
Cursor  move to next box once first box has a number in it.
Keyboard dismiss when tap  on the screen.
Slide down to dismiss the screen.
screen is designed using StoryBoard.

**How to test this**
click on the button in main storyboard

**ScreenShots & Gifs**:

leezit7

### CODE REVIEW

**Review by RiffatiKhan — APPROVED** [review]
Looks ok to me

**Review by RiffatiKhan — DISMISSED** [review]
Looks fine

File: LeezeIt/LeezeIt/VerificationViewController.swift [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            case textFieldThree :
+                textFieldFour.becomeFirstResponder()
+                break
+            case textFieldFour :
+                textFieldFour.resignFirstResponder()
+                break
+            default :
+                break
+            }
+        } else {}
```
Comment (haroonmaqsoodch) [inline LeezeIt/LeezeIt/VerificationViewController.swift]: remove else if it's not needed

File: LeezeIt/LeezeIt/VerificationViewController.swift [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    }
+    override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
```
Comment (haroonmaqsoodch) [inline LeezeIt/LeezeIt/VerificationViewController.swift:59]: add a space before override func

File: LeezeIt/LeezeIt/VerificationViewController.swift [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        self.view.endEditing(true)
+    }
+    
+    func setup() {
+        textFieldOne.addTarget(self, action: #selector(textDidChange(textField:)), for: UIControl.Event.editingChanged)
+        textFieldTwo.addTarget(self, action: #selector(textDidChange(textField:)), for: UIControl.Event.editingChanged)
+        textFieldThree.addTarget(self, action: #selector(textDidChange(textField:)), for: UIControl.Event.editingChanged)
+        textFieldFour.addTarget(self, action: #selector(textDidChange(textField:)), for: UIControl.Event.editingChanged)
+    }
+       }
```
Comment (haroonmaqsoodch) [inline LeezeIt/LeezeIt/VerificationViewController.swift]: fix the indentation

File: LeezeIt/LeezeIt/VerificationViewController.swift [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+import UIKit
+
+class VerificationViewController: UIViewController,UITextFieldDelegate {
+
+    @IBOutlet weak var textFieldOne: UITextField!
+    @IBOutlet weak var textFieldTwo: UITextField!
+    @IBOutlet weak var textFieldThree: UITextField!
+    @IBOutlet weak var textFieldFour: UITextField!
+    @IBOutlet weak var veriefyButton: UIButton! {
```
Comment (haroonmaqsoodch) [inline LeezeIt/LeezeIt/VerificationViewController.swift]: fix spellings "verify"

File: LeezeIt/LeezeIt/VerificationViewController.swift (hunk 3 above)
Comment (Harrisali93) [reply LeezeIt/LeezeIt/VerificationViewController.swift]: resolved

File: LeezeIt/LeezeIt/VerificationViewController.swift (hunk 4 above)
Comment (Harrisali93) [reply LeezeIt/LeezeIt/VerificationViewController.swift]: resolved

File: LeezeIt/LeezeIt/VerificationViewController.swift (hunk 2 above)
Comment (Harrisali93) [reply LeezeIt/LeezeIt/VerificationViewController.swift:59]: resolved

File: LeezeIt/LeezeIt/VerificationViewController.swift (hunk 1 above)
Comment (Harrisali93) [reply LeezeIt/LeezeIt/VerificationViewController.swift]: resolved

**Review by haroonmaqsoodch — DISMISSED** [review]

**Review by RiffatiKhan — DISMISSED** [review]
seems alright

**Review by arfhan786 — APPROVED** [review]

**Review by RiffatiKhan — APPROVED** [review]
Looks fine

**Pull request merged**